### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,19 +1,25 @@
-= Mobile Docs Previewer
+= Mobile Docs Previewer (docs.aerogear.org)
 
-Allows you preview proposed changes to docs.aerogear.org without installing antora.
+Allows you preview a static html page with proposed changes to docs.aerogear.org without installing Antora.
```
Comment (pwright) [inline README.adoc]: a static website?

File: README.adoc [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 
 . Fork this repo.
-. Edit the site.yml to point to your fork and the branch you want to preview, for example:
+. Edit the site.yml to point to your forks of aerogear/mobile-docs and aerogear/mobile-security and specify the branchyou want to preview, for example:
```
Comment (pwright) [inline README.adoc]: and/or
you probably are only changing one of these

File: README.adoc [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 +
 ----
   - url: https://github.com/finp/mobile-docs.git
     branches: fix-things
+    
+  - url: https://github.com/finp/mobile-security.git
+    branches: fix-things
```
Comment (pwright) [inline README.adoc]: is this making it too complex, maybe we should limit the scope of this repo to only mobile-docs, and remove mobile-security from site.yml?

File: README.adoc [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
 ----
-. Commit and push your changes to master branch.
-. Create a netlify account if you don't already have one at https://www.netlify.com/
-. Create a *New site from Git* in netlify.
-. Select the fork you created in step one.
+. Edited `site.yml` must be in the `master` branch.
```
Comment (pwright) [inline README.adoc]: not a task

File: README.adoc [hunk 5]
```diff
@@ -1,19 +1,25 @@
...
+. Create a link:https://www.netlify.com[^Netlify account] if you don't already have one.
+. Click *New site from Git* in Netlify.
+. Select the fork of `mobile-netlify` you created in step one.
 . Select to use the master branch.
 . Click *Deploy*.
+. First deploy can take up to 10 minutes. Subsequent deploys will take ~30 seconds.
```
Comment (pwright) [inline README.adoc]: my experience is that subsequent deploys can also take many minutes

**Review by pwright — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 3 above)
Comment (pwright) [reply README.adoc]: keeping with both repos for now

File: README.adoc (hunk 5 above)
Comment (ghost) [reply README.adoc]: 40 seconds for removed content

Comment (ghost) [reply README.adoc]: solved

File: README.adoc (hunk 1 above)
Comment (ghost) [reply README.adoc]: resolved

File: README.adoc (hunk 2 above)
Comment (ghost) [reply README.adoc]: resolved

File: README.adoc (hunk 3 above)
Comment (ghost) [reply README.adoc]: well, imo better to keep both

resolved

File: README.adoc (hunk 4 above)
Comment (ghost) [reply README.adoc]: resolved

**Comment by pwright** [comment]
@pmacko1 I'd like to update readme, but haven't seen issues re. this PR resolved.

**Comment by ghost** [comment]
@finp https://github.com/finp/mobile-netlify/pull/2/files ?

**Pull request merged**